main.sh

```diff
@@ -34,12 +34,14 @@ declare -r BIN_DIR="$PROJECT_ROOT/tmp/bin"
 # PROMETHEUS_ENABLE=${PROMETHEUS_ENABLE:-false}
 # CONFIG_OUT_DIR=${CONFIG_OUT_DIR:-"/tmp/generated-manifest"}
 #
+
+# shellcheck disable=SC1091
```

Comment:
I noticed that in this PR, there are different styles of shellcheck annotation among different shell files, so what's the best practice and guide for such annotations then?

Reply:
In this particular case, we know that SC1091 which checks if the file actually exists need not be done because 
`[[ -f ... ]]` does that already